proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -119,9 +129,9 @@ message Vote {
   option (gogoproto.goproto_stringer) = false;
   option (gogoproto.equal)            = false;
 
-  uint64     proposal_id = 1 [(gogoproto.moretags) = "yaml:\"proposal_id\""];
-  string     voter       = 2;
-  VoteOption option      = 3;
+  uint64                      proposal_id  = 1 [(gogoproto.moretags) = "yaml:\"proposal_id\""];
+  string                      voter        = 2;
+  repeated WeightedVoteOption options      = 3 [(gogoproto.nullable) = false];
```

Comment:
I'd like to make this less breaking for clients.

I can think of two approaches. Either keep both and use whichever one was used during voting:
```suggestion
  // option will be set to VOTE_OPTION_UNSPECIFIED if weighted_options are used
  VoteOption option = 3;
  // weighted_options will be empty if option is used
  repeated WeightedVoteOption weighted_options      = 4 [(gogoproto.nullable) = false];
```

or deprecate the old field:
```suggestion
  reserved 3;
  reserved "option";
  repeated WeightedVoteOption options      = 4 [(gogoproto.nullable) = false];
```

Reply:
Would this mean I don't have to move it to v1beta2?

Reply:
Yeah I think so

Reply:
Made the change.  The CI protobuf-breakage check still fails though.  It says:
```
proto/cosmos/gov/v1beta1/gov.proto:128:1:Previously present field "3" with name "option" on message "Vote" was deleted.
```

Is this okay?